e2e/wdio/core/tests/grid-list.e2e-spec.ts

```diff
@@ -124,4 +137,249 @@ describe('Grid-list test suite', function() {
+                checkElementStates(gridListItemError, itemTag + '12-', item);
+            });
+
+            it('verify link states', () => {
+                scrollIntoView(gridListLink, 42);
+                checkElementStates(gridListLink, linkTag + '12-', link, 42);
+            });
+        });
+
+        describe('Check status examples', function() {
+            it('verify item states', () => {
+                const itemStatusLength = getElementArrayLength(gridListItemStatus);
+                for (let i = 0; i < itemStatusLength; i++) {
+                    scrollIntoView(gridListItemStatus, i);
+                    checkElementStates(gridListItemStatus, itemTag + i + '-', item, i);
+                }
+            });
+
+            it('verify link states', () => {
+                const linkStatusLength = getElementArrayLength(gridListLinkStatus);
+                for (let i = 0; i < linkStatusLength; i++) {
+                    scrollIntoView(gridListLinkStatus, i);
+                    checkElementStates(gridListLinkStatus, linkTag + i + '-', link, i);
+                }
+            });
+        });
+
+        describe('Check more button example', function() {
+            it('verify item states', () => {
+                scrollIntoView(gridListItem, 48);
+                checkElementStates(gridListItem, itemTag + '13-', item, 48);
+            });
+
+            it('verify link states', () => {
+                scrollIntoView(gridListLink, 52);
+                checkElementStates(gridListLink, linkTag + '13-', link, 52);
+            });
+        });
+
+        describe('Check footer example', function() {
+            it('verify item states', () => {
+                scrollIntoView(gridListItem, 53);
+                checkElementStates(gridListItem, itemTag + '14-', item, 53);
+            });
+
+            it('verify link states', () => {
+                scrollIntoView(gridListLink, 57);
+                checkElementStates(gridListLink, linkTag + '14-', link, 57);
+            });
+        });
+
+        describe('Check drag and drop example', function() {
+            it('verify item states', () => {
+                scrollIntoView(gridListItem, 58);
+                checkElementStates(gridListItem, itemTag + '15-', item, 58);
+            });
+
+            it('verify link states', () => {
+                scrollIntoView(gridListLink, 62);
+                checkElementStates(gridListLink, linkTag + '15-', link, 62);
+            });
+        });
+
+        describe('Check layout example', function() {
+            it('verify item states', () => {
+                scrollIntoView(gridListItem, 66);
+                checkElementStates(gridListItem, itemTag + '16-', item, 66);
+            });
+
+            it('verify link states', () => {
+                scrollIntoView(gridListLink, 70);
+                checkElementStates(gridListLink, linkTag + '16-', link, 70);
+            });
+        });
+
```

Comment:
Where are the functional cases?
Need to check:
multiselect allows multiple selections
single select only allows 1 item to be selected
drag and drop works
more btn shows more cards
delete btn removes card
etc.